### DISCUSSION TITLE
How to set the Dialog Service in wpf to have the same Background Mask as the Dialog Service in Xamarin.Forms?

### DISCUSSION THREAD

**Discussion by apchenjun** [PrismLibrary/Prism#3022] [WPF] [answered]
I want to set the Dialog Service in wpf to have the same Background Mask effect as the Dialog Service in Xamarin.Forms.
![image](https://github.com/PrismLibrary/Prism/assets/4964551/f5d19b70-10d7-4d9f-a998-aa80970d5f7f)

**Comment by brianlagunas** [comment, marked as answer]
That's not how dialogs work in WPF. If you want this behavior, you'll need to manually add a mask layer to your view and change it's color/transparency when you show/hide the dialog. Or, you need to use a dialog control that isn't based on WPF Windows, but rather just a content control you embed in your view.
